DB/src/report/report.controller.js

```diff
@@ -54,4 +54,14 @@ router.put('/reports/:repId/severity_score/:empId', async (req,res) => {
   res.json(response);
 });
 
+router.get('/reports/:repId/comments', async (req,res) => {
+  //if(notLoggedIn(req, res)) return;
+
+  let{connection, message} = await conn.getConnection(res);
+  if (message == 'fail') return;
+
+  let response = await model.getComments(connection, req.params.repId);
+  res.json({theComment: response.comments});
```

Comment:
In this section, be sure to return the whole response, now that it has both a message for succeed or fail, and the payload. I go into specifics in `report.model.js`.